spanner/cloud-client/src/test/resources/header.csv

```diff
@@ -0,0 +1,3 @@
+int,float,bool,string,date,timestamp
```

Comment:
I'm a bit confused, I thought headers are column names, but here they seem they are types?

Reply:
Sorry, yeah this is somewhat confusing. The test case here to test the different supported types used the column names as type names - will be modified.